src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ProjectPropertiesProviders/DelegatedProjectPropertiesBase.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.ProjectPropertiesProviders
+{
+    /// <summary>
+    /// An implementation of IProjectProperties that delegates its operations
+    /// to another IProjectProperties object
+    /// </summary>
+    internal class DelegatedProjectPropertiesBase : IProjectProperties
+    {
+        protected internal IProjectProperties DelegatedProperties;
```

Comment:
Why "protected internal"? Why not just a public property if its needed outside of the class?